mrbgems/mruby-print/src/print.c

```diff
@@ -13,7 +13,7 @@ printstr(mrb_state *mrb, mrb_value obj)
 {
   if (mrb_string_p(obj)) {
 #if defined(_WIN32)
-    if (isatty(fileno(stdout))) {
+    if (_isatty(_fileno(stdout))) {
```

Comment:
I'm not familiar with mingw, but underbar'ed names are real names in MSVCRT, aren't them?
Then I fell we should use them when using them does not make problem with other platforms.